clang/lib/Driver/ToolChains/ClassicFlang.h

```diff
@@ -0,0 +1,49 @@
+//===--- ClassicFlang.h - Flang ToolChain Implementations -------*- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_LIB_DRIVER_TOOLCHAINS_Flang_H
+#define LLVM_CLANG_LIB_DRIVER_TOOLCHAINS_Flang_H
```

Comment:
Shouldn't this be ..._ClassicFlang_H ?

Reply:
This commit should be in this port.

Reply:
The commit is not in this port because it is already part of the baseline llvmorg-14.0.1 (commit 26f56438e3dab44cea4c8f16d4cb16e9424b02c6).

Reply:
The macro has been fixed.